library/std/src/lib.rs

```diff
@@ -456,6 +456,7 @@ pub use core::pin;
 #[stable(feature = "rust1", since = "1.0.0")]
 pub use core::ptr;
 #[stable(feature = "rust1", since = "1.0.0")]
+#[allow(deprecated_in_future)]
```

Comment:
I think it wants this to be:

```suggestion
#[allow(deprecated)]
```

given that 1.53.0 is already the present as far as nightly is concerned. Otherwise it's currently failing with:

```console
error: use of deprecated module `core::raw`: use pointer metadata APIs instead https://github.com/rust-lang/rust/issues/81513
   --> library/std/src/lib.rs:460:9
    |
460 | pub use core::raw;
    |         ^^^^^^^^^
    |
    = note: `-D deprecated` implied by `-D warnings`
```

Reply:
It looks like stage0 (compiling from beta) emits `deprecated_in_future` and stage1+ emits `deprecated`